.github/workflows/ci.yml

```diff
@@ -28,12 +23,42 @@ jobs:
     - uses: actions/setup-dotnet@v1
       with:
         dotnet-version: '5.0.100'
-        
+  
     - run: ./build.sh build -s true
       name: Build
+
     - run: ./build.sh generatepackages -s true
       name: Generate local nuget packages
+
     - uses: actions/upload-artifact@v2
       with:
         name: nuget-packages
         path: build/output/*
+
+    - run: ./build.sh validatepackages -s true
+      name: Validate *.npkg files that were created
+
+    - name: Publish packages to GitHub package repository
+      if: github.event_name == 'push' && startswith(github.ref, 'refs/heads/main')
+      shell: bash
+      run:  |
+         OWNER=$GITHUB_REPOSITORY_OWNER
```

Comment:
Is this for readability of the subsequent line?

Reply:
Exactly. This way it won't need to be updated when we merge to upstream.